Sources/PaylikeSDK/Views/Composites/ExtendedWhitelabelPaymentForm.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  ExtendedWhitelabelPaymentForm.swift
+//
+//
+//  Created by Székely Károly on 2023. 05. 10..
+//
+
+import SwiftUI
+import PaylikeClient
+
+struct ExtendedWhitelabelPaymentForm: View {
+    @State private var email: String = "";
```

Comment:
These state variables are defined here in the View because the VM is not created yet for the ExtendedWhiteLabelPaymetnForm, right?

Reply:
Thats correct :)